### ISSUE TITLE
aio -h command does not exist

### ISSUE THREAD

**Issue by alexdkaminski** [AdobeDocs/adobe-io-runtime#87]

**Comment by purplecabbage** [comment]
the 2 prs resolve this issue, please merge them and close this

**Comment by alexdkaminski** [comment]
I don't have write access so cannot merge the PR

**Comment by purplecabbage** [comment]
Yeah, sorry @alexdkaminski I should have been more clear. The message was intended for someone who does have write access ( I don't either )
Thank you for your commits!

**Comment by purplecabbage** [comment]
Done, thanks!
